indicator/textchannelobserver.cpp

```diff
@@ -499,6 +499,9 @@ void TextChannelObserver::showNotificationForMessage(const Tp::TextChannelPtr ch
             messageText = QString::fromUtf8(C::ngettext("Attachment: %1 audio clip", "Attachments: %1 audio clips", audioCount)).arg(audioCount);
         } else if (attachmentCount > 0) {
             messageText = QString::fromUtf8(C::ngettext("Attachment: %1 file", "Attachments: %1 files", attachmentCount)).arg(attachmentCount);
+        } else if (attachmentCount == 0) {
```

Comment:
You have before attahcmentCount > 0, so now the elseif can be just else or? There will be no sane way that attachmentcount is smaller than 0 :) -so please make it unconditional else only.